R/utils_object.R

```diff
@@ -84,5 +84,5 @@ object_has_rownames <- function(x) {
     stop("Only dataframe objects are allowed.", call. = FALSE)
   }
 
-  !all(is.numeric(attributes(x)$row.names))
+  !identical(attributes(x)$row.names, 1:nrow(x))
```

Comment:
```suggestion
  !identical(attributes(x)$row.names, seq_along(x))
```

``` r
1:nrow(data.frame())
#> [1] 1 0

seq_along(data.frame())
#> integer(0)
```

<sup>Created on 2022-09-30 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

Reply:
I used `seq_len(nrow(x))` instead, like the original code (doesn't change perf)